### PR TITLE
[C-1862] Add back button to manual profile creation screen

### PR DESCRIPTION
**Pull request by sliptype**
### Description
Simulator Screen Shot - iPhone 14 Pro - 2023-01-18 at 17 00 46

We were missing the back button on this screen on mobile

### Dragons

Touches sign up navigation! Had to switch most of the navigation actions to push in order to get the replace animation to be a pop

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by dylanjeffers — COMMENTED** [review]
This is great, real quick, does this also add back/x button on the screen right after inputting email? we can get into stuck states there where you maybe make a mistake and want to go back to "sign-in" screen.

**Comment by sliptype** [comment]
> This is great, real quick, does this also add back/x button on the screen right after inputting email? we can get into stuck states there where you maybe make a mistake and want to go back to "sign-in" screen.

No it doesn't, this was just to get parity with mobile web. Mobile web doesn't have a back button there either. I definitely think we should add it, but let me make a separate ticket and run it past product

File: packages/mobile/src/screens/signon/FirstFollows.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const FirstFollows = (props: FirstFollowsProps) => {
       })
     )
 
-    navigation.replace('SignupLoadingPage')
+    navigation.push('SignupLoadingPage')
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/signon/FirstFollows.tsx:52]: nice

File: packages/mobile/src/screens/signon/SignupHeader.tsx [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
-const SignupHeader = () => {
+const SignupHeader = (props: SignupHeaderProps) => {
+  const { showBackButton, onPressBack } = props
+  const styles = useStyles()
+  const { neutralLight4 } = useThemeColors()
   return (
-    <View style={{ backgroundColor: 'white', zIndex: 15 }}>
+    <View style={styles.container}>
       <View style={styles.header}>
+        {showBackButton ? (
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/signon/SignupHeader.tsx:54]: nice

**Review by dylanjeffers — APPROVED** [review]
great add

**Pull request merged**